cores/esp32/Stream.h

```diff
@@ -60,6 +60,24 @@ class Stream: public Print
 
     void setTimeout(unsigned long timeout);  // sets maximum milliseconds to wait for stream data, default is 1 second
     unsigned long getTimeout(void);
+    
+    bool find(char *target);   // reads data from the stream until the target string is found
+    bool find(uint8_t *target) { return find ((char *)target); }
+    // returns true if target string is found, false if timed out (see setTimeout)
+
+    bool find(char *target, size_t length);   // reads data from the stream until the target string of given length is found
+    bool find(uint8_t *target, size_t length) { return find ((char *)target, length); }
+    // returns true if target string is found, false if timed out
+
+    bool find(char target) { return find (&target, 1); }
+
+    bool findUntil(char *target, char *terminator);   // as find but search ends if the terminator string is found
+    bool findUntil(uint8_t *target, char *terminator) { return findUntil((char *)target, terminator); }
+
+    bool findUntil(char *target, size_t targetLen, char *terminate, size_t termLen);   // as above but search ends if the terminate string is found
+    bool findUntil(uint8_t *target, size_t targetLen, char *terminate, size_t termLen) {return findUntil((char *)target, targetLen, terminate, termLen); }
+
+/*    
```

Comment:
same here about dead code and retaining the const signature updates.

Reply:
I leave this code to Espressif decide what to do. About const it was not there on original AVR Core code.

Reply:
unless the code modifies the chars, they should be const. Regardless if that is on AVR or not :)